mmcv/runner/hooks/lr_updater.py

```diff
@@ -139,8 +139,9 @@ def get_lr(self, runner, base_lr):
 
 class PolyLrUpdaterHook(LrUpdaterHook):
 
-    def __init__(self, power=1., **kwargs):
+    def __init__(self, power=1., min_lr=0.,  **kwargs):
```

Comment:
There are two whitespaces, while only one is expected.

Reply:
It is revised.